js-exercises/array-cube-root-to-json/arrayCubeRootToJson.js

```diff
@@ -0,0 +1,63 @@
+function getCube(number) {
+  return number * number * number;
+}
+
+function isEveryElementInteger(array) {
+  return array.every(elem => {
+    if (elem === Infinity || elem === -Infinity) {
+      return true;
+    }
+    elem = parseInt(elem);
+    return !Number.isNaN(elem);
+  });
+}
+
+function getCubeRoot(number) {
```

Comment:
Good to see that you have written a custom function for finding **cube root**. You could have used `Math.cbrt` function though, in this case as there was no need for this. 
Anyways good work.